docs/source/customizing.rst

```diff
@@ -1,6 +1,63 @@
 Customizing your BinderHub deployment
 =====================================
 
+Frontend
+--------
+
+Header and Footer customization
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+BinderHub uses `Jinja <https://jinja.palletsprojects.com/en/stable/>`_ as template engine
+to process the page template `binderhub/templates/page.html <https://github.com/jupyterhub/binderhub/blob/main/binderhub/templates/page.html>`.
```

Comment:
There's something wrong with the link syntax- it's not linked:
https://binderhub--1923.org.readthedocs.build/en/1923/customizing.html#header-and-footer-customization

Reply:
I think it'd be helpful to also highlight the important parts of the template- is `<div id="root"></div>` the critical bit?